src/components/Dashboard/FilingHistoryList/bodies/FutureEffectivePending.vue

```diff
@@ -61,6 +61,12 @@ export default class FutureEffectivePending extends Vue {
         filingLabel: 'dissolution'
       }
     }
+    if (EnumUtilities.isTypeContinuationIn(this.filing)) {
+      return {
+        subtitle: 'Incorporation Pending',
+        filingLabel: 'continuatin in'
```

Comment:
Typo here Sev.

Reply:
Yes, also a unit test failure that I'll fix in next commit...

Reply:
Both are fixed.

Reply:
Thanks. Resolving.